package.json

```diff
@@ -140,6 +140,7 @@
     "jquery.iframe-transport": "1.0.0",
     "jquery.transition": "artsy/jquery.transition",
     "jsdom": "11.6.2",
+    "jwt-decode": "^3.0.0-beta.2",
```

Comment:
Just curious (non-blocking): Are we depending on something in this beta over the stable `2.2.0` version?

Reply:
Oh, no... It's just what got installed by default. I'll update to the 2.2.0